python/py-numpy/Portfile

```diff
@@ -96,7 +96,6 @@ if {${name} ne ${subport}} {
                             -gcc46 -gcc47 -gcc48 -g95
 
     depends_lib-append      port:fftw-3 \
-                            port:py${python.version}-nose \
```

Comment:
I believe that this dependency is required for older NumPy used for older Python ... we need to test this out & if so then add it to the dependency list for those.

Reply:
right, I didn't check that sorry... but on the other hand I find it hard to believe that `nose` would actually be anything other than a test dependency. Since there is no test-phase defined in the Portfile it shouldn't be needed. 

Of course, if the intention here is to allow people to do `import numpy ; numpy.test()` then it could be needed. In that case it should make sure to have all test dependencies added (I had a missing `hypothesis` either here or in scipy). But that would benefit from a comment in the Portfile or we could just leave this up to the user...

Reply:
good points! I guess I need a "test" phase then ... since I do that after updating and before pushing the updates ... just to make sure testing does something useful.